### PR TITLE
[Doc Improvement][Implement-guidelines-for-images-Build-first-app-using-SPFx][2307242]

### PR DESCRIPTION
**Pull request by v-silakshmi**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-spfx.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/a0c824d0f6c54300a6f104e08e4a0828c3be33a0/msteams-platform/sbs-gs-spfx.yml) | 95 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a09bbeed-3a1d-43d4-923a-e26c3eaea5e4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-spfx.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/4fda318be95f53eba9c46261b3145fc029de7500/msteams-platform/sbs-gs-spfx.yml) | 95 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ed494551-3d4c-4287-a11e-7533c7ed682e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-spfx.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/0b77dfc74d34e39d547861ed4206a324fd578cdd/msteams-platform/sbs-gs-spfx.yml) |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-spfx.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/9f03742af40613a581cbbd1b191d887e88b1d72a/msteams-platform/sbs-gs-spfx.yml) | 95 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9b30df01-a930-411d-8e38-8fe146043dc1) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-spfx.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/0a42ffa92951c0ab30060c3643740610d4a45f10/msteams-platform/sbs-gs-spfx.yml) | 95 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fbd3e5bd-1840-4997-8e2e-243efdfa451c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-spfx.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/8f8c112067f554acae5a6b80c4be1ccf094ce6a1/msteams-platform/sbs-gs-spfx.yml) | 95 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2d5f9cbf-9616-450b-b741-8c5a65eb8fd5) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/sbs-gs-spfx.yml [hunk 1]
```diff
@@ -293,7 +293,7 @@ items:
         teamsfx account login m365
         ```
 
-        Your default web browser opens to let you sign in to the account. Sign in to your account using your credentials. Close the browser when you're prompted.
```
Comment (ashishguptaiitb) [inline msteams-platform/sbs-gs-spfx.yml:296]: The first instance with full stop reads better to me. Check once with other writers.

**Comment by ashishguptaiitb** [comment]
- We should remove the browser address bar, assuming it is redundant. If not, then retain. 
- `hw-teams-toolkit-sidebar-icon1.png`, just add the icon instead of the whole screenshot.
- `ssl-prompt.png` seems like a redundant screenshot. If not necessary, drop it.

Sharing some updates for actioning. CC @v-silakshmi , @v-habibaumme

File: msteams-platform/sbs-gs-spfx.yml (hunk 1 above)
Comment (v-silakshmi) [reply msteams-platform/sbs-gs-spfx.yml:296]: Hi Ashish, reverted to original instance.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-spfx.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/54ee3c0b48ef7ecfbfd7d733d7fbf2263073103d/msteams-platform/sbs-gs-spfx.yml) | 95 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fdb21ea3-4656-40f3-8bf9-ba947b474914) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-spfx.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/6ee18760aac043b253407620b69f7654ae4f5c81/msteams-platform/sbs-gs-spfx.yml) | 95 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/542559e7-e459-4d1f-a4c4-2274c979e8fc) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by ashishguptaiitb — APPROVED** [review]
lgtm

**Pull request merged**